MaxSukhanov/school_docs
# add authentication with google

**What's new here?**
Now User can log in via google

**How to check it out?**

1. Go to `http://localhost:3000/users/sign_in`
2. Click on the link `Login with Google`
3. Select account or insert your credential in the Google form
4. Check the email (via mailcatcher or console) to confirm account. Then you will be redirected to the login page
5. Insert your credentials and sing in
